### PR TITLE
compiler-rt 15.0.4

### PR DESCRIPTION
**Pull request by h-vetinari**
Update #50 for main

Closes #50 
Closes #54

### CODE REVIEW

**Comment by h-vetinari** [comment]
CC @isuruf in case you want to review

This is +/- exactly the state of your last commit into #50, merged with main & updated to 15.0.4

File: recipe/bld.bat [hunk 1]
```diff
@@ -21,11 +20,11 @@ cmake ^
 if %ERRORLEVEL% neq 0 exit 1
 
 :: Build step
-nmake VERBOSE=1
+cmake --build .  --target compiler-rt
```
Comment (xhochy) [inline recipe/bld.bat:23]: Could the build error come from the fact that we are building a specific target here? At least the logs suggest that the required `stats.cpp` (the only source of `clang_rt.stats`) source file is never compiled.

Comment (h-vetinari) [reply recipe/bld.bat:23]: Without the target, nothing at all gets compiled, see the CI of the previous commit, resp. https://github.com/llvm/llvm-project/issues/58893

**Comment by h-vetinari** [comment]
Interestingly, it works with nmake... 🤔

**Pull request merged**